dependencies/graknlabs/dependencies.bzl

```diff
@@ -69,8 +69,8 @@ def graknlabs_simulation():
 def graknlabs_verification():
     git_repository(
         name = "graknlabs_verification",
-        remote = "https://github.com/graknlabs/verification",
-        commit = "7ebb27486bb333b15dee0ca46e547276d0368e37",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_verification
+        remote = "https://github.com/alexjpwalker/verification",
```

Comment:
lets not forget to bump this back before merging!

Reply:
done